conjur/errors.py

```diff
@@ -69,3 +69,21 @@ class CredentialRetrievalException(Exception):
     def __init__(self):
         self.message = FETCH_CREDENTIALS_FAILURE_MESSAGE
         super().__init__(self.message)
+
+class KeyringGeneralError(Exception):
+    """ Exception for when configuration is invalid """
```

Comment:
these have the same comments so I am not sure what each is doing and when.

Reply:
ctrl-c exception :-)
Thnx